HiP-ThumbnailService/Utility/EndpointConfig.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace PaderbornUniversity.SILab.Hip.ThumbnailService.Utility
+{
+    public class EndpointConfig
+    {
+
+        public string HostUrl { get; set; }
```

Comment:
Hm... I think `HostUrl` shouldn't belong to `EndpointConfig`, since it's more a thumbnail-config thing (like `Path`). It doesn't specify an endpoint where another service or database can be reached. Also, please document whether this should end with a slash or not (because it looks like it matters).